CONTRIBUTING.md

```diff
@@ -2,6 +2,6 @@
 
 We're so happy that you want to contribute to OpenPBS!
 
-Please see the Contributor's Portal for details, guidelines, and how-to tutorials.  Start at **[Becoming a Contributor to PBS Pro](https://pbspro.atlassian.net/wiki/spaces/DG/pages/20414474/Becoming+a+Contributor+to+PBS+Pro)**.
+Please see the Contributor's Portal for details, guidelines, and how-to tutorials.  Start at **[Becoming a Contributor to OpenPBS](https://pbspro.atlassian.net/wiki/spaces/DG/pages/20414474/Becoming+a+Contributor+to+PBS+Pro)**.
```

Comment:
I believe the url could also be changed to openpbs.atlassian.net (from pbspro.atlassian.net)

Reply:
This is pending for a while, guess we will take it as it is anyway.